torch/_decomp/decompositions.py

```diff
@@ -3235,15 +3235,17 @@ def affine_grid_generator(theta: Tensor, size: List[int], align_corners: bool):
         return _affine_grid_generator_5d(theta, size, align_corners=align_corners)
 
 
-@register_decomposition(aten.grid_sampler_2d)
-@pw_cast_for_opmath
-def grid_sampler_2d(
+def _grid_sampler_2d(
```

Comment:
Leave a comment here or where we register the decomp saying that we leave this here in case other backends want to use it, and because it computes the "correct" strides, while the inductor decomp doesn't.